mem-06-j.xml

```diff
@@ -76,12 +76,12 @@
       <column name="definition_de">Datenübertragung</column>
       <column name="definition_fa">انتقال داده [AUTOTRANSLATED]</column>
       <column name="definition_sv">dataöverföring</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">передавать данные</column>
       <column name="definition_zh_HK">數據傳輸 [AUTOTRANSLATED]</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{QIn:n:1}, {lab:v}, {lI':v:2}, {nav QIn:n}</column>
-      <column name="notes"></column>
+      <column name="notes">Это слово часто используется, чтобы сослаться на отправку Email, среди Клингонистов</column>
```

Comment:
This shouldn't be in the "notes" field. The "hidden_notes" field shouldn't be translated at all, as it relates to things which are not properly about the language, but meta-knowledge about the language, like puns and such. Anyone who cares about such things are assumed to speak English, since most of the information wouldn't make sense otherwise.